packages/web/src/common/store/pages/signon/reducer.js

```diff
@@ -104,8 +104,7 @@ const actionsMap = {
       // even if toggling b/w sign up and sign in redirects to the right place
       routeOnCompletion: state.routeOnCompletion,
       routeOnExit: state.routeOnExit,
-      isMobileSignOnVisible: state.isMobileSignOnVisible,
-      followArtists: state.followArtists
```

Comment:
Currently we seem to be hanging onto artists but I cant figure out why, might need to ask around before merging this. I cant come up with a good reason tbh, this is specifically in the `signOn` part of the store so it shouldn't be needed elsewhere.
Last commit touching this was "open source audius" lol

Reply:
I agree we can clear this. Maybe it was needed to follow artists at some other point? But with out new setup not needed